src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/ReflectMethods.java

```diff
@@ -1005,7 +1033,7 @@ public void visitIdent(JCIdent tree) {
                         }
                     }
                 }
-                case INTERFACE, CLASS, RECORD, ENUM -> {
+                case PACKAGE, INTERFACE, CLASS, RECORD, ENUM -> {
```

Comment:
This is a pre-existing issue -- e.g. calling `println` with a fully qualified name, like `java.lang.System.out.println` causes an exception, because we eventually visit `java` and `java.lang` which are packages